src/test/kotlin/uk/gov/justice/digital/hmpps/personrecord/model/PersonTest.kt

```diff
@@ -28,63 +29,12 @@ internal class PersonTest {
     assertThat(person.middleNames).containsExactlyInAnyOrder("Jack", "Michael", "Henry")
   }
 
-  @Test
```

Comment:
These have been disabled for months so I thought I would tidy up as I am in this file anyway